<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrectPost</class>
 <widget class="QWidget" name="CorrectPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>853</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="topLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Изменить объявление&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="imageViewPub">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>192</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>192</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="toAddImage">
        <property name="text">
         <string>Добавить картинку</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="productName">
        <property name="maximumSize">
         <size>
          <width>590</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="aboutTextField">
        <property name="maximumSize">
         <size>
          <width>590</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberTextField">
        <property name="maximumSize">
         <size>
          <width>590</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Номер телефона</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="publicationButton">
      <property name="maximumSize">
       <size>
        <width>590</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Публикация</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Удалить публикацию</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
